CONFIG.md

```diff
@@ -478,12 +479,52 @@ from the project directory when building a package. The above includes README
 files and files under bin/ in the project's package.
 
 **packages.dest.docroot**: Specify where within the package directory the `srcFiles`
-should be placed. Defaults to the package root. For Acquia set this to
-'/docroot'.
+should be placed. Defaults to the package root. For Acquia set this to '/docroot'.
 
 **packages.dest.devResources**: Specify where within the package directory the
 `projFiles` should be placed. Defaults to package root.
 
+
+### Project Operations
+
+Some of the most common use cases for any Drupal site are the basic operations
+of installation, running database updates, and clearing the cache. These are not
+difficult operations, but many developers appreciate the common shorthand a
+little grunt magic can provide.
+
+To keep this flexible the new Operations system offers nothing out of the box,
+but facilitates creating lightweight commands or callouts to the shell for more
+advanced scripts you want easily run on the project by the entire team.
+
+This should remind you of the **Themes Scripts** described above.
+
+```
+{
+  "scripts": {
+    "install": "drush <%= config.alias %> install <%= config.profile %> -yv"
+    "update": "drush <%= config.alias %> updatedb -yv && drush <%= config.alias %> features-revert-all -yv"
+  }
+}
+```
+
+Here you see two commands defined inside the top-level `scripts` configuration.
+`grunt install` demonstrates a straightforward installation of the project's
+profile. `grunt update` demonstrates running database updates and reverting all
+[Features](http://www.drupal.org/project/features). GDT does not ship either of
+these, though we are exploring how best to initialize a project with sensible
+default operations via [Gadget](https://github.com/phase2/generator-gadget).
+
+The Alias and Profile is specified per the rules for **Magic Configuration**.
+
+**alias**: The Drush Site Alias. This is computed as a helper for use in
+project oeprations, but is not yet applied as part of other tasks. Defaults to
+`@<domain>`. The environment variable is `GDT_SITE_ALIAS`.
+
+**profile**: The Drupal installation profile, this will be used to configure the
+behavior of the drush:liteinstall task. Defaults to `standard`.
+WARNING: drush:liteinstall is an internal task and is likely to be deprecated in
```

Comment:
Let's remove this warning. We're offering another use case for the variable, and it doesn't seem in the spirit of semver.